lmbdtest/test_downsample_volume_lambda.py

```diff
@@ -15,19 +15,55 @@
 # lambdafcns contains symbolic links to lambda functions in boss-tools/lambda.
 # Since lambda is a reserved word, this allows importing from that folder 
 # without updating scripts responsible for deploying the lambda code.
-from lambdafcns.downsample_volume import (
-    S3DynamoDBTable, downsample_volume, S3Bucket
-)
 
-from bossutils.multidimensional import XYZ
-import numpy as np
+import importlib
+# from lambda import downsample_volume as dsv
+dsv = importlib.import_module("lambda.downsample_volume")
```

Comment:
👍 Didn't think about using this to get around the unfortunate lambda folder name.